ci/common.sh

```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 #
 
+# shellcheck disable=SC2148
```

Comment:
I think it is quite strange to add a warning and disable it later. Let's remove it.

Reply:
SC2148(Add a shebang) rule has enabled, remove it `ci code-lint` will run failed. 
The key point is that, all the scripts have the shebang but `ci/common.sh` and `t/cli/common.sh`.

`disable` in script file, just scope the file, no influences on the other.

To fix the check failure, three solutions to choose:

1. enable SC2148 and add shellcheck disable declaration(just like now), test case [SC2148](https://github.com/kwanhur/apisix/runs/5301576428?check_suite_focus=true).
2. enable SC2148 and add a shebang in `common.sh`, but it seems not so suitable, like https://github.com/apache/apisix/pull/6340#discussion_r809653666.
3. disable SC2148, this will not find it when add a script without shebang.

Summary, solution 1 could be the better.
Hope your voice.

Reply:
Sure, but why use

> enable=SC2148

in https://github.com/apache/apisix/pull/6428/files#diff-c71502ae2fb6ee770263dbf93031c6bb1549ef6b2970b862daf24d01a1ded277R22?

Could we keep it disabled? Look like the check is enabled by you.

Reply:
Had disable it.

Actually, all the rules enable by default. Here, I mark it obviously.